sources/platform/console/two-factor-authentication.md

```diff
@@ -0,0 +1,97 @@
+
+Under the recovery codes, you will find two fields for your recovery information. These two fields are what the support team will ask you to provide in case you lose access to your authenticator app and also to your recovery codes. We will never use the phone number for anything else than to verify your identity and help you regain access to your account and only as a last resort. Ideally, the personal information provided by you will be enough to verify your identity. Always provide both what kind of personal information you provide and the actual information.
+
+:::info Personal information
+What kind of personal information you provide is completely up to you, it does not even have to be personal, as long as it's secure and easy to remember. For example, it can be the name of your pet, the name of your favorite book, some secret code, or anything else. Keep in mind who has access to that information. While you can use the name of your pet, if you share information about your pet on public social media, it's not a good choice, because anyone on the internet can access it. The same goes for any other information you provide.
+:::
+
+You will not be able to enable the two-factor authentication until you click on the **Download** / **Copy** buttons or copy the codes manually. After you do that, the **Continue** button will light up and you can click on it to enable the two-factor authentication. The authentication process will then enable the two-factor authentication for your account and show a confirmation.
+
+### Confirmation
+
+When you close the setup process, you should see that your two-factor authentication is enabled in the account settings.
+
+![Apify console two-factor authentication section enabled](./images/console-account-two-factor-enabled.png)
+
+
+## Verification after sign-in
+
+After you enable two-factor authentication, the next time you attempt to sign in, you'll need to enter a code before you can get into the Apify Console. To do that, open your authenticator app and enter the code for your Apify account into the **code** field. After you enter the code, click on the **Verify** button and if the provided code is correct, you will proceed to Apify Console.
+
+![Apify console two-factor authentication form](./images/console-two-factor-authentication.png)
+
+## Using recovery codes
+
+In case you lose access to your authenticator app, you can use the recovery codes to sign in to your account. To do that, click on the **recovery code or begin 2FA account recovery** link below the **Verify** button. This will redirect you to a similar view as the current one, but instead of code from the authenticator app, you will need to enter one of the 16 recovery codes that you have received during the setup process.
```

Comment:
```suggestion
In case you lose access to your authenticator app, you can use the recovery codes to sign in to your account. To do that, click on the **recovery code or begin 2FA account recovery** link below the **Verify** button. This will redirect you to a view similar to the current one, but instead of code from the authenticator app, you will need to enter one of the 16 recovery codes you received during the setup process.
```